control/controlclient/direct.go

```diff
@@ -104,7 +104,7 @@ type Direct struct {
 // Observer is implemented by users of the control client (such as LocalBackend)
 // to get notified of changes in the control client's status.
 type Observer interface {
-	SetControlClientStatus(Status)
+	SetControlClientStatus(Client, Status)
```

Comment:
kinda gross, but understandable. gross enough to need docs now, though. it's not entirely obvious.

Reply:
done